src/transformers/models/prophetnet/modeling_prophetnet.py

```diff
@@ -1956,7 +1970,7 @@ def forward(
             >>> import torch
 
             >>> tokenizer = ProphetNetTokenizer.from_pretrained('microsoft/prophetnet-large-uncased')
-            >>> model = ProphetNetForCausalLM.from_pretrained('patrickvonplaten/prophetnet-decoder-clm-large-uncased')
+            >>> model = ProphetNetForCausalLM.from_pretrained('microsoft/prophetnet-large-uncased')
```

Comment:
much much  better to use the "real" models instead of heaving to tweak the "real" model to make a new "same" model that is compatible with `ProphetNetForCausalLM`